### DISCUSSION TITLE
Pybamm developer guide

### DISCUSSION THREAD

**Discussion by krishnakantsingh14** [pybamm-team/PyBaMM#2336] [General]
Hi PyBamm team and Community,

   I am looking for a developer guide of pybamm to expand it further. Please let me know, if there is any documentation available.  

Thanks,
Krishna

**Comment by valentinsulzer** [comment]
We have contributing info here: https://github.com/pybamm-team/PyBaMM/blob/develop/CONTRIBUTING.md
Otherwise there isn't much, ask about specific things for help. Issues labeled "good first issue" are good ones to start with to get familiar with the codebase
